www-src/app/change/change_manager.coffee

```diff
@@ -0,0 +1,43 @@
+FileManager = require "./file_manager"
+log = require('../lib/persistent_log')
+    prefix: "Change Manager"
+    date: true
+
+###*
+ * ChangeManager allows to launch the good manager with specific state
+ *
+ * @class ChangeManager
+###
+module.exports = class ChangeManager
```

Comment:
Interesting. I think I do the same with word helper. When I don't know how to name a bunch of functions.